src/main/java/lotto/controller/LottoGame.java

```diff
@@ -0,0 +1,24 @@
+package lotto.controller;
+
+import lotto.domain.LottoGroup;
+import lotto.domain.Money;
+import lotto.domain.Ranks;
+import lotto.domain.WinningLotto;
+import lotto.ui.InputView;
+import lotto.ui.ResultView;
+
+import java.math.BigDecimal;
+
+public class LottoGame {
+    public void start() {
+        Money money = new Money(InputView.getMoney());
+        LottoGroup lottoGroup = LottoGroup.buyAutoSelectLotto(money.getCountOfLotto());
+        ResultView.printLottoGroup(lottoGroup.getLottos());
+        String winningLottoNumbers = InputView.getWinningLotto();
+        int bonusNo = InputView.getBonusNo();
+        WinningLotto winningLotto = new WinningLotto(winningLottoNumbers, bonusNo);
```

Comment:
재사용하지 않는 지역 변수의 수는 최소화하는 것이 좋아요.

```suggestion
        WinningLotto winningLotto = new WinningLotto(InputView.getWinningLotto(), InputView.getBonusNo());
```